BGL/doc/BGL/PackageDescription.txt

```diff
@@ -1,33 +1,193 @@
 /// \defgroup PkgBGL CGAL and the Boost Graph Library Reference
 
-/// \defgroup PkgBGLConcepts Concepts
-/// \ingroup PkgBGL
-///
-/// We extend the Boost Graph Library(\sc{Bgl} for short) with a set of new concepts. 
-/// The documentation of the concepts lists at the same time the functions
-/// related to it. Models of the concept and their related functions
-/// must be in the same namespace (they will be found by Koenig lookup).
-///
-///\dot
-///digraph example {
-///  node [shape=record, fontname=Helvetica, fontsize=10];
-///  rankdir=BT
-///
-///  "boost Graph" [ URL="http://www.boost.org/libs/graph/doc/Graph.html" ];
-///  HalfedgeGraph [ URL="\ref HalfedgeGraph"];
-///  HalfedgeListGraph [ URL="\ref HalfedgeListGraph"];
-///  MutableHalfedgeGraph [ URL="\ref MutableHalfedgeGraph"];
-///  FaceGraph [ URL="\ref FaceGraph"];
-///  FaceListGraph [ URL="\ref FaceListGraph"];
-///  MutableFaceGraph [ URL="\ref MutableFaceGraph"];
-///  
-///  MutableHalfedgeGraph -> HalfedgeGraph -> "boost Graph" [ arrowhead="open", label="refines" ];
-///  FaceListGraph -> FaceGraph -> HalfedgeGraph [ arrowhead="open", label="refines" ];
-///  MutableFaceGraph -> MutableHalfedgeGraph [ arrowhead="open", label="refines" ];
-///  MutableFaceGraph -> FaceGraph [ arrowhead="open", label="refines" ];
-///  HalfedgeListGraph -> HalfedgeGraph [ arrowhead="open", label="refines" ];
-///}
-///\enddot
+/*! \defgroup PkgBGLConcepts Concepts
+ \ingroup PkgBGL
+
+ We extend the Boost Graph Library (\sc{Bgl} for short) with a set of new concepts. 
```

Comment:
Actually `EdgeListGraph` is already a concept in the BGL. I guess you want to mention it to get links to free functions, but in that case you better mention `VertexAndEdgeListGraph` to also get vertex related functions.